keyboards/westm/westm68/rules.mk

```
@@ -0,0 +1,25 @@
+# MCU name
+MCU = STM32F072
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+DFU_SUFFIX_ARGS = -v FFFF -p FFFF
+
+# Build Options
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```